ads/adstir.md

```diff
@@ -14,23 +14,37 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# AdStir
+# adstir
 
 ## Example
 
+### banner
+
 ```html
 <amp-ad width="320" height="50"
     type="adstir"
-    data-app-id="MEDIA--343ded3e"
+    data-app-id="MEDIA-343ded3e"
     data-ad-spot="1">
 </amp-ad>
 ```
 
+### native
+
+```html
+<amp-ad width="310" height="280"
+    type="adstir"
+    data-app-id="MEDIA-343ded3e"
+    data-ad-spot="4"
+    data-ad-type="native">
+</amp-ad>
+```
+
 ## Configuration
 
-For configuration details and to generate your tags, please refer to [your publisher account](https://ad-stir.com/login) or [contact AdStir](https://ad-stir.com/contact).
+For configuration details and to generate your tags, please refer to [your publisher account](https://ad-stir.com/login) or [contact adstir](https://ad-stir.com/contact).
 
 Supported parameters:
 
 - `data-app-id`
 - `data-ad-spot`
+- `data-ad-type`
```

Comment:
Please update the doc, and indicate that `data-app-id` and `data-ad-spot` are required, while the `data-ad-type` is optional. You can refer to our [example doc](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/amp-ad.md#supported-ad-networks).